src/explorer/siteNodeBase.ts

```diff
@@ -197,46 +188,42 @@ export class SiteNodeBase extends NodeBase {
         // the scm url lives in the 1 index of enabledHostNames, not 0
         const remote = `https://${username}:${password}@${repo}`;
 
-
-        let git = require('simple-git/promise')(workspace.rootPath);
-
+        const git = require('simple-git/promise')(workspace.rootPath);
         try {
-            let status = await git.status();
+
+            const status = await git.status();
             if (status.files.length > 0) {
                 window.showWarningMessage(`There ${status.files.length > 1 ? 'are' : 'is'} ${status.files.length} uncommitted change${status.files.length > 1 ? 's' : ''} in local repo "${workspace.rootPath}"`);
             }
-            await git.push(remote, 'master');
-        }
-        catch (err) {
+            await git.push(remote, 'HEAD:master');
+        } catch (err) {
             if (err.message.indexOf('spawn git ENOENT') >= 0) {
-                let input = await window.showErrorMessage(`Git must be installed to use Local Git Deploy.`, `Install`)
+                const input = await window.showErrorMessage(`Git must be installed to use Local Git Deploy.`, `Install`);
```

Comment:
Declare a const string for "Install" and use the const, easier for localization.